src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -21,6 +21,19 @@ nginx_sites:
         # If the header is not provided, do not rate limit
         - key: '""'
           value: '""'
+    # Remove 'username' query param from request uri for use in cache key
+    # https://stackoverflow.com/a/45587134
+    # https://stackoverflow.com/a/35772678
+    - name: "$request_uri_no_username"
+      variable: "$request_uri"
+      default_value: "$request_uri"
+      var_mappings:
```

Comment:
I wasn't able to get a single regex to work for all combinations of param placement so just went with 2 versions + a default fall through